components/layout/navbar/navbar.tsx

```diff
@@ -101,6 +101,7 @@ const Navbar = ({ DesHeight }) => {
           align-items: center;
           box-shadow: 0px 16px 32px rgba(0, 0, 0, 0.25);
           width: 100%;
+          top: 0;
```

Comment:
Why do you add the "top: 0"?

Reply:
the navbar wasn’t sticking to the top of the screen but now I see it is acting fine so I will remove it

Reply:
removed!